src/etc/systemd/system/ocserv.service.d/override.conf

```diff
@@ -0,0 +1,14 @@
+[Unit]
+RequiresMountsFor=/run
+ConditionPathExists=/run/ocserv/ocserv.conf
+After=
+After=vyos-router.service
+After=dbus.service
```

Comment:
Does it depend on dbus? Then you should add an additional line `Requires=dbus.service`

Reply:
`After=dbus.service` exisist even in native ocserv.service.